pkg/notifier/email.go

```diff
@@ -0,0 +1,53 @@
+package notifier
+
+import (
+	"fmt"
+	"github.com/go-mail/mail"
+	"github.com/krzysztof-gzocha/curnot/pkg/config"
+)
+
+const NameEmailNotifier = "email"
+
+type Email struct {
+	dialer               *mail.Dialer
```

Comment:
I think we wouldn't like to anyhow modify `mail.Dialer` parameters from notifier perspective. Is it really required to have it as a pointer? We will have just 1 email notifier, right? Which imply that we will have only 1 `mail.Dialer`, right? Changing it from pointer to struct would incur that `Email` struct will receive a copy of a struct - impossible to change global state. Sounds a bit better, correct? Similar stuff with config params